src/modules/airline/index.ts

```diff
@@ -0,0 +1,237 @@
+/**
+ * IATA stands for [International Air Transport Association](https://iata.org).
+ * It's the trade association for the world's airlines and it is
+ * responsible for setting standards relating to many aspects of airline
+ * operations.
+ */
+import type { Faker } from '../..';
+
+export enum Aircraft {
+  Narrowbody = 'narrowbody',
+  Regional = 'regional',
+  Widebody = 'widebody',
+}
+
+export type AircraftType = `${Aircraft}`;
+
+export interface Airline {
+  /**
+   * The name of the airline (e.g. `'American Airlines'`).
+   */
+  readonly name: string;
+  /**
+   * The 2 character IATA code of the airline (e.g. `'AA'`).
+   */
+  readonly iataCode: string;
+}
+
+export interface Airplane {
+  /**
+   * The name of the airplane (e.g. `'Airbus A321'`).
+   */
+  readonly name: string;
+  /**
+   * The IATA code of the airplane (e.g. `'321'`).
+   */
+  readonly iataTypeCode: string;
```

Comment:
```suggestion
  /**
   * The IATA code of the airplane (e.g. `'321'`).
   */
  readonly iataTypeCode: string;
```

What's the difference between an `iataCode` and an `iataTypeCode`

Should `Type` be reflected in the JSDoc?

Reply:
`iataTypeCode` is what I should have named the field to begin with, as that's a more descriptive name. I've also updated the JSDocs to match, thanks for catching that!